autopush/router/apns2.py

```diff
@@ -21,6 +22,16 @@
 APNS_PRIORITY_LOW = '5'
 
 
+class ComplexEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, Decimal):
+            return int(float(obj.to_integral()))
```

Comment:
Hrm, ok, so reading the docs and thinking about this, there are a few possible methods that I should take. While the outcome is "dump this out as an integer" and this is absolutely temporary while the iOS folk are poking at it, there's a few things at play. 

`to_integral` (or properly `to_integral_value`) does the rounding inside the decimal without flagging an exception. float converts the decimal to a numeric and int converts to the expected JSON serializable element. Having decimal do it's rounding is probably the most accurate thing. The other two steps are pretty much required.